flask/config.py

```diff
@@ -5,11 +5,12 @@
 
 class Config(object):
     SQLALCHEMY_DATABASE_URI = os.getenv('DATABASE_URL', 'postgresql:///photo2code_dev')
-    DEBUG = False
-    TESTING = False
+    DEBUG = True
+    TESTING = True
     CSRF_ENABLED = True
-    SECRET_KEY = os.urandom(24)  ##TODO: Change this!!
-
+    ##TODO: Change this!!
+    SECRET_KEY = 'app secret'
```

Comment:
100% change this when we actually deploy. Kept as this for local development and testing.